src/renderer/ColorManager.ts

```diff
@@ -112,9 +112,9 @@ export class ColorManager implements IColorManager {
   public setTheme(theme: ITheme): void {
     this.colors.foreground = this._parseColor(theme.foreground, DEFAULT_FOREGROUND);
     this.colors.background = this._parseColor(theme.background, DEFAULT_BACKGROUND);
-    this.colors.cursor = this._parseColor(theme.cursor, DEFAULT_CURSOR);
-    this.colors.cursorAccent = this._parseColor(theme.cursorAccent, DEFAULT_CURSOR_ACCENT);
-    this.colors.selection = this._parseColor(theme.selection, DEFAULT_SELECTION);
+    this.colors.cursor = this._parseColor(theme.cursor, DEFAULT_CURSOR, true);
```

Comment:
Shouldn't the cursor use `this.allowTransparency`?

Reply:
The cursor is drawn on a separate transparent render layer, so it should be possible to use a transparent cursor without `allowTransparency`, similar to how you can do a transparent selection.